dotnet/hello_world/HelloWorldSettings.cs

```diff
@@ -1,4 +1,23 @@
-﻿namespace HelloWorld
+﻿/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is not the correct header, we did not license this code to the ASF and the ASF does not license it to the users. This header is used only in ASF-managed projects; typically, Apache 2.0 license projects outside of ASF do not reference ASF explicitly in the header or license.

As I mentioned, the correct header can be generated via https://github.com/mbrukman/autogen; it looks like this:

```
// Copyright 2017 Google Inc.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Please use this format for all the code in this repo.

Reply:
To install autogen for Windows users a minor modification in the instructions is needed

cd /bin
git clone https://github.com/mbrukman/autogen.git
echo 'alias autogen=/bin/autogen/autogen' >> ~/.bash_profile
source ~/.bash_profile

in Windows
cd ~ --> HOME directory --> /c/Users/UserName
cd / --> directory where Git is installed --> C:\Program Files\Git